illumos-utils/src/addrobj.rs

```diff
@@ -5,7 +5,17 @@
 //! API for operating on addrobj objects.
 
 /// The name provided to all link-local IPv6 addresses.
-pub const IPV6_LINK_LOCAL_NAME: &str = "ll";
+pub const IPV6_LINK_LOCAL_ADDROBJ_NAME: &str = "ll";
+
+/// The name provided to all static IPv6 underlay addresses.
+pub const IPV6_STATIC_ADDROBJ_NAME: &str = "omicron6";
+
+/// The name provided to all static IPv4 addresses, usually for public OPTE
+/// interfaces.
+pub const IPV4_STATIC_ADDROBJ_NAME: &str = "omicron6";
+
+/// The name provided to DHCP-configured addresses, of either family.
+pub const DHCP_ADDROBJ_NAME: &str = "omicron";
```

Comment:
Nit: These constants are only used in `running_zone.rs`, perhaps they should be moved there for readability? Either way, I'm not too fussed about this, if you disagree feel free to ignore this comment :)

Reply:
We have the link-local addrobj suffix `ll` here already. Since these are also address-object naming conventions, I think it makes more sense to keep them here.